clang/TranslationUnit.d

```diff
@@ -74,6 +77,16 @@ struct TranslationUnit
         return DiagnosticVisitor(cx);
     }
 
+    @property DiagnosticSet diagnosticSet ()
+    {
+        return DiagnosticSet(clang_getDiagnosticSetFromTU(cx));
+    }
+
+    @property size_t numDiagnostics ()
```

Comment:
Is this used because it's more efficient than creating a `DiagnosticSet` and call `length`?